src/main/java/seedu/duke/commons/util/TimeUtil.java

```diff
@@ -0,0 +1,55 @@
+package seedu.duke.commons.util;
+
+import seedu.duke.commons.core.Messages;
+import seedu.duke.commons.util.exceptions.TimeParseException;
+
+public class TimeUtil {
+    /**
+     * Parses 12-hour clock format to 24-hour clock format.
+     *
+     * @param clock 12-hour clock
+     * @return 24-hour clock
+     * @throws TimeParseException if input clock is invalid
+     */
+    public static String parseTwelveHourClock(String clock) throws TimeParseException {

```

Comment:
Just for the sake of consistency, I suggest that "time" as a better word than "clock" for the method and variable naming, since the class is called "TimeUtil".